Fix the following error:
llvm_propeller_mock_whole_program_info.h:7:10: fatal error:
'llvm_propeller_cfg.pb.h' file not found.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 173d711..3f72d2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,7 @@ add_dependencies(llvm_propeller_options quipper_perf)
 add_library(llvm_propeller_mock_whole_program_info OBJECT
 	    llvm_propeller_mock_whole_program_info.cc)
 add_dependencies(llvm_propeller_mock_whole_program_info llvm_propeller_options)
+add_dependencies(llvm_propeller_mock_whole_program_info llvm_propeller_cfg_proto)
 add_library(create_llvm_prof_object OBJECT create_llvm_prof.cc)
 add_dependencies(create_llvm_prof_object llvm_propeller_options)
 
